lib/snapshot/BUILD.gn

```diff
@@ -389,3 +392,36 @@ generate_vm_patched_sdk("flutter_patched_sdk") {
     ],
   ]
 }
+
+action("compile_platform") {
+  script = "//dart/tools/compile_platform.py"
+
+  sources = [
+    "$root_out_dir/flutter_patched_sdk/lib/libraries.json",
+  ]
+
+  outputs = [
+    "$root_out_dir/flutter_patched_sdk/platform.dill",
```

Comment:
shouldn't this be vm_platform.dill so it matches what the VM expects?

Reply:
VM(reusable between standalone Dart and Flutter Engine) does not have an opinion on file name. 
Dart standalone embedder(the one that uses dfe.cc) does, but flutter engine is separate embedder and it still expects `platform.dill`(https://github.com/flutter/engine/blob/master/runtime/dart_init.cc#L70) as well as flutter tools does (https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/artifacts.dart#L48 and https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/flx.dart#L30).

Reply:
Ah! thanks for clarifying. LGTM

In the future, it might be nice to rename the file then to flutter_platform and update the embedder to expect that. It is not necessary, but it could make it easier to understand error messages and to debug issues if they every arise.